tools/astexplorer/src/index.tsx

```diff
@@ -75,20 +79,53 @@ document.addEventListener('DOMContentLoaded', async () => {
     }
   );
 
-  const parser = await CadenceParser.create("cadence-parser.wasm")
+  const root = createRoot(astElement)
+  const render = root.render.bind(root)
```

Comment:
you can use `root::render` with @babel/plugin-proposal-function-bind. but we also don't use babel for the ast explorer 😅